src/shell/IRGenerator.cpp

```diff
@@ -104,6 +104,18 @@ export class IRGenerator final: public CoreVM::IRBuilder, public ast::Visitor
         _result = createCallFunction(getBuiltinFunction(node.callback.get()), callArguments, "set");
     }
 
+    void visit(ast::VariableSubstExpr const& node) override
+    {
+        auto callArguments = std::vector<CoreVM::Value*> {};
+        if (node.name )
+        {
+
+            callArguments.push_back(codegen(node.name.get()));
+        }
+
+        _result = createCallFunction(getBuiltinFunction(node.callback.get()), callArguments, "get");
```

Comment:
```suggestion
        _result = createCallFunction(getBuiltinFunction(node.callback.get()), callArguments, "getenv");
```
as mentioned above, either `getenv` or `variableSubst` :)

Reply:
Renamed to getenv